TOOLS-2473 bump jenkins-agent major version to 3

timfoster opened the pull request:
I added testing notes to the ticket.

timfoster commented:
I've only added that to the experimental channel in the past, but have imported them into all of the staging DCs. It's no problem to promote to all channels if needed (does stuff auto-expire from experimental?)

trentm commented:
> does stuff auto-expire from experimental?

Practically, I don't *believe* so (but I could be forgetting). Theoretically there is no promise to keep stuff that is in experimental forever. At the *least* I'd have them in "dev" if they are being used.

timfoster commented:
Cool. I've done that.
```
$ updates-imgadm channel-add   dev 3e8e972d-1d38-4b68-a3b2-00861b2ec89a
Added image 3e8e972d-1d38-4b68-a3b2-00861b2ec89a (jenkins-agent-i386-1.6.3@3.0.0) to "dev" channel
$ updates-imgadm channel-add   dev f00a393d-a1c8-4e93-a13e-c364a7d0a2a9
Added image f00a393d-a1c8-4e93-a13e-c364a7d0a2a9 (jenkins-agent-multiarch-15.4.1@3.0.0) to "dev" channel
$ updates-imgadm channel-add   dev d639b789-ce4e-4c81-a230-a28172dd0360
Added image d639b789-ce4e-4c81-a230-a28172dd0360 (jenkins-agent-x86_64-18.4.0@3.0.0) to "dev" channel
$ updates-imgadm channel-add   dev e6539349-b85c-4401-92ef-ea4dad86c2bc
Added image e6539349-b85c-4401-92ef-ea4dad86c2bc (jenkins-agent-x86_64-19.4.0@3.0.0) to "dev" channel
```
